src/components/Book.js

```diff
@@ -1,20 +1,66 @@
 import React /* , { useEffect } */ from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 import removeBookData from '../redux/books/removeBookAPI';
 
 const Book = ({
   id, title, author, category,
 }) => {
   const dispatch = useDispatch();
   const delBook = () => dispatch(removeBookData(id));
-
+  const percentage = 66;
   return (
     <div className="book-div">
-      <p>{category}</p>
-      <h3>{title}</h3>
-      <p>{author}</p>
-      <button type="button" onClick={delBook}>Remove</button>
+      <div className="book-data">
+        <div className="category">{category}</div>
+        <div className="title">{title}</div>
+        <div className="author">{author}</div>
+        <ul className="book-footer-links">
+          <li><NavLink className="footer-link">Comments</NavLink></li>
+          <li className="space-li"> | </li>
+          <li><NavLink className="footer-link" onClick={delBook}>Remove</NavLink></li>
+          <li className="space-li"> | </li>
+          <li><NavLink className="footer-link">Edit</NavLink></li>
+        </ul>
```

Comment:
- [ ] @emmanuellmhango great work on your styling, however the color in author and comments section differs to the color given check the comparison below:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/208178030-fbca76bf-067a-45d0-8b3e-36839598413e.png) |  ![image](https://user-images.githubusercontent.com/61408860/208178143-fb400adb-818c-467c-9a9d-580c255a0ca1.png)

Reply:
Well resolved 👍🏽

Reply:
- [ ] [REQUIRED] Well done @emmanuellmhango , I see that the changes you made was just changing the color. Kindly note that the last reviewer recommended that you change the arrangement of the content in your project navbar. I have checked your design and found this:
 
<img width="1312" alt="Screenshot 2022-12-16 at 22 08 29" src="https://user-images.githubusercontent.com/58692590/208189752-9c6dee86-1f28-4cfa-bae9-4c6a6927148b.png">

Kindly restyle it to be like the Zeplin design
![1](https://user-images.githubusercontent.com/58692590/208189927-79ba2b8f-ce96-4dca-a08e-025025409415.png)

Reply:
Well resolved 👍🏽

Reply:
Thank you. @Whoistolu 

![navbar](https://user-images.githubusercontent.com/40168702/208191090-e2f685b5-8319-4524-8236-50c6792fccd4.JPG)

Reply:
Great implementation @emmanuellmhango !